todolist/app.go

```diff
@@ -137,6 +137,25 @@ func (a *App) ExpandTodo(input string) {
 	fmt.Println("Todo expanded.")
 }
 
+func (a *App) ManipulateNotes(input string) {
+	a.Load()
+	id := a.getId(input)
+	parser := &Parser{}
+	todo := a.TodoList.FindById(id)
+	if todo == nil {
+		fmt.Println("No such id.")
+		return
+	}
+
+	retStr := parser.ParseNotes(todo, input)
```

Comment:
I think this should be refactored.  `ParseNotes` violates command query separation since it manipulates notes.  I'd rather see a separate function on line 151 where we can ask the parser if a note was added/modified/deleted in the previous command.  Maybe something like `if parser.NotesWereChanged(todo, input)` or something like that.   Then you can run a.Save() and user another function like `parser.NoteUserOutput()` to show the output to the user, if any.

Also, I think renaming this function to something like `handleNotes` makes more sense.  At first glance, when I see `ParseNotes` I automatically think it will return some sort of parsed note.  In reality, this function should return nothing.

Reply:
I was also able to make it say `Notes deleteed` with the way you're concatenating the action for the user output.

Reply:
How about dividing current `ParseNotes` into the four kinds of sub-command: `ParseAddNote`, `ParseShowNote`, `ParseDeleteNote`, `ParseEditNote`?  And in such modification, we can follow the same pattern as `ParseEditTodo`.